### ISSUE TITLE
[Suggestion] Would be nice to see water values in item tooltips, as well as add/change custom values

### ISSUE THREAD

**Issue by vizthex123** [Charles445/SimpleDifficulty#25]
Title. Would be noice.

**Comment by Charles445** [comment]
I'll have to look into adding tooltips, that'd be handy

You can change custom values of drinks and stuff, there's commands in-game (try /sd help) and json files in the config/simpledifficulty folder

**Comment by vizthex123** [comment]
Ah, ok. Didn't know there were commands.

I would prefer an example config with some directions though, if possible.

**Comment by Charles445** [comment]
Yeah I really need to get around to proper documentation on stuff, ideally when things get ready for a proper release (when all assets and features are in and functional)

**Comment by vizthex123** [comment]
noice, would be good. This is one of the main mods in my 1.12.2 modpack, and it's a bit annoying how you can't see the thirst value for items.

also, could you add support to [Classic Bars](https://www.curseforge.com/minecraft/mc-mods/classic-bars)? The mod dev said he's abandoned it.
